src/Sylius/Component/Media/spec/Model/ImageSpec.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Component\Image\Model;
+
+use PhpSpec\ObjectBehavior;
+
+/**
+ * @author Aram Alipoor <aram.alipoor@gmail.com>
+ */
+class ImageSpec extends ObjectBehavior
```

Comment:
I'm not sure, but it looks like there should be something more than one method :)